virttest/shared/cfg/base.cfg

```diff
@@ -1088,6 +1088,9 @@ uuid_dimm = ""
 # Set image_extent_size_hint, displayed in qemu-img info extend_size_hint field.
 #image_extent_size_hint = 1G #To set extent_size_hint to 1G
 
+#image_compression_type, which is used for a qcow2 image(create or convert) only
```

Comment:
Would you mind formatting the comments, e.g.  (please polish the wording to make it more clear from your perspective)
# Set the compression type when creating or converting a qcow2 image
# Supported types: zlib, zstd
#image_compression_type = zstd

The last line just gives an example, without a space between '#' and 'image', so it's easy for someone to uncomment the line and make the param effective.